src/python/zquantum/core/circuit/conversions/pyquil_conversions.py

```diff
@@ -50,18 +52,32 @@
 TWO_QUBIT_CONTROLLED_GATES = {
     CZ: pyquil.gates.CZ,
     CNOT: pyquil.gates.CNOT,
-    SWAP: pyquil.gates.SWAP,
 }
 
 
+PYQUIL_NAME_TO_ORQUESTRA_CLS = {
```

Comment:
I'm assuming this is meant to just be a "reversed" mapping compared to `SINGLE_QUBIT_NONPARAMETRIC_GATES`, `ROTATION_GATES`, and `TWO_QUBIT_CONTROLLED_GATES`? If so, let's just add a comment here explaining the relationship.